.changeset/nine-suits-jump.md

```diff
@@ -0,0 +1,5 @@
+---
+'create-keystone-next-app': patch
+---
+
+Fixed a windows issue where CKA would not run via npx or npm init
```

Comment:
People are seeing this in the changelog for this package, they already know what package you're talking about and CKA isn't really accurate sort of?

```suggestion
Fixed a windows issue where the package would not run via npx or npm init
```